tfworker/commands/terraform.py

```diff
@@ -117,27 +125,19 @@ def exec(self):
                 )
                 raise SystemExit(2)
 
-            if self._force_apply:
+            if not execute:
+                click.secho(f"no plan changes for {definition.tag}", fg="yellow")
+
+            if self._force and (self._tf_apply or self._destroy):
                 execute = True
+                click.secho(f"forcing {self._plan_for} due to --force", fg="red")
 
-            if execute and self._tf_apply:
-                if self._force_apply:
-                    click.secho(
-                        f"force apply for {definition.tag}, applying",
-                        fg="yellow",
-                    )
-                else:
-                    click.secho(
-                        f"plan changes for {definition.tag}, applying",
-                        fg="yellow",
-                    )
-            elif execute and self._destroy:
-                click.secho(
-                    f"plan changes for {definition.tag}, destroying",
-                    fg="yellow",
-                )
-            elif not execute:
-                click.secho(f"no plan changes for {definition.tag}", fg="yellow")
+            # there are no plan changes, and no forced execution, so move on
+            if not execute:
+                continue
+
+            # no apply/destroy requested, so move on
+            if not self._tf_apply and not self._destroy:
```

Comment:
Is it harmless to pass both apply and destroy flags? e.g. `worker --apply --no-destroy` or `worker --apply --destroy`?

Reply:
The class will raise a systemexit duriung init if you attempt to both apply and destroy in the same run, it would not work right as the ordering of definitions is generated only once in memory, and differs between apply/destroy.

Reply:
in the first example `--apply --no-destroy` would set apply to true and destroy to false, by default both apply and destroy are false, so that is the same effect as passing only `--apply`